content/en/chaos_engineering.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Chaos Engineering or CE is the discipline of experimenting on a [distributed system](https://glossary.cncf.io/distributed_systems/) in production to build confidence in the system's capability to withstand turbulent and unexpected conditions.
+
+## Problem it addresses
+People working in the field of [SRE](https://glossary.cncf.io/site_reliability_engineering/) and [DevOps](https://glossary.cncf.io/devops/) continuously look for new techniques to increase product resiliency and [reliability](https://glossary.cncf.io/reliability/). A system's ability to tolerate failures while ensuring adequate service quality is typically a software development requirement. But deep testing on a production system is not always possible due to the complexity of the architecture. High-frequency deployment of new features to the production environment can result in a high probability of downtime and a critical incident — with considerable consequences to the business.
```

Comment:
Adding the info about experiments and improving the wording helped, thanks @fsbaraglia. Still I think we can improve: 

"But deep testing on a production system is not always possible due to the complexity of the architecture" I am not quite sure if this is entirely true, and more important if this is really the problem that Chaos Engineering tries to solve. 

What about replacing the sentence with something like this: "There are several aspects involved that could lead to outages of an application, like infrastructure, network or other moving parts of a (microservice-based) application. High-frequency deployment**s** to such an environment could result in a downtime ..."

Reply:
good one, thank you @iamNoah1